### PR TITLE
DB storage of canvas developer keys

### PR DESCRIPTION
**Pull request by lbjay**
- New model: CanvasDeveloperKey
- enable admin site
- make CanvasApiToken.user a foreign key into user model

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,20 +24,9 @@ Setup
 
     url(r'^canvas_api_token/', include('canvas_api_token.urls'))
 
-3. Add a ``LTI_APP_DEVELOPER_KEYS`` entry to your settings that looks like this:
+3. Run `python manage.py migrate` to ensure db tables are initialized.
+4. Use the admin site to create a `cavnas_dev_key` entry using the `consumer_key` and developer key values from your Canvas consumer where ...
```
Comment (djcp) [inline README.rst]: sp - "cavnas"

File: README.rst [hunk 2]
```diff
@@ -24,20 +24,9 @@ Setup
...
-        '[oauth-consumer-key]': {
-            'client_id': '[client_id]',
-            'client_secret': '[client_secret]'
-        }
-    }
-
-where ...
-
-* ``oauth_consumer_key`` value is the consumer key used when registering your LTI tool in the Canvas account admin
 * ``client_id`` is the integer client id value of your Canvas `developer key <https://canvas.instructure.com/doc/api/file.oauth.html>`_
```
Comment (djcp) [inline README.rst:30]: errant underscore at EOL?

File: HISTORY.rst [hunk 3]
```diff
@@ -18,3 +18,12 @@ Release History
...
 * added unique index to CanvasApiToken.user_id
+
+0.2.0 (2015-04-28)
+++++++++++++++++++
+
+DB storage of canvas developer keys
+
+* New model: CanvasDeveloperKey
+* enable admin site
+* make CanvasApiToken.user a foreign key into user model
```
Comment (djcp) [inline HISTORY.rst:29]: Is there code that must be run to logically migrate old records into the new code beyond just modifying the database?  If so, or if this breaks stuff (and there are users other than yourself), it'd be nice to document that.

Comment (lbjay) [inline HISTORY.rst:29]: Good catch. This was me being lazy and assuming I'm the only user. Which is no doubt true, but still. I should do it the right way with django migrations.

File: README.rst (hunk 2 above)
Comment (lbjay) [inline README.rst:30]: No, that's rst link syntax (which is admittedly strange). http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets

**Pull request merged**